documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -271,6 +272,31 @@ To explicitly preserve the case of the table and field names that are present in
 When this options is set, when an incoming event is for a topic called `orders`, and the destination database dialect is Oracle, the connector creates a table with the name `orders`, because the constructed SQL defines the name of the table as `"orders"`.
 Enabling quoting results in the same behavior when the connector creates column names.
 
+[[jdbc-connection-idle-timeouts]]
+=== Connection Idle Timeouts
+
+The JDBC sink connector for {prodname} leverages a connection pool to enhance performance.
+Connection pools are engineered to establish an initial set of connections, maintain a specified number of connections, and efficiently allocate them to the application as required.
+However, a challenge arises when connections linger idle in the pool, potentially triggering timeouts if they remain inactive beyond the configured idle timeout threshold of the database.
+
+To mitigate this issue, connection pools offer a mechanism to periodically validate the connection's activity.
+This validation ensures that connections remain active and prevent them from being perceived as idle by the database.
+In case of a network disruption, attempting to utilize a terminated connection prompts the pool to replace it with a fresh one.
```

Comment:
```suggestion
In the event of a network disruption, if {prodname} attempts to use a terminated connection, the connector prompts the pool to replace it with a fresh one.
```
Is it Debezium/the connector the prompts for a new connection?

Reply:
Yes, _connector_ is right, _connection_ was incorrect.